Build/GameInitOptionsScreen.cc

```diff
@@ -309,7 +316,13 @@ static void EnterGIOScreen()
 	{ // JA2Gold: iron man buttons
 		INT16  const x   = GIO_IRON_MAN_SETTING_X + GIO_OFFSET_TO_TOGGLE_BOX;
 		INT16  const y   = GIO_IRON_MAN_SETTING_Y - GIO_OFFSET_TO_TOGGLE_BOX_Y;
-		size_t const def = gGameOptions.fIronManMode ? GIO_IRON_MAN : GIO_CAN_SAVE;
+		size_t def;
+		switch (gGameOptions.ubGameSaveMode) {
+		  	case GIO_CAN_SAVE:   	def = GIO_CAN_SAVE; break;
+			default:
+			case GIO_IRON_MAN: 	def = GIO_IRON_MAN;  break;
+			case GIO_DEAD_IS_DEAD:  def = GIO_DEAD_IS_DEAD; break;
+		}
```

Comment:
Why not just:
size_t const def = gGameOptions.ubGameSaveMode ? gGameOptions.ubGameSaveMode : GIO_CAN_SAVE;

... or not allowing for it to ever be unset?